requirements.txt

```diff
@@ -0,0 +1,5 @@
+reno~=2.11.2
```

Comment:
Reno and sphinx bits aren't needed if we aren't doing docs in the repo. I don't know about the need for yq or not

Reply:
Agreed